### PR TITLE
feat(helm_generate): enable staging of generated params

### PR DESCRIPTION
**Pull request by vdice**

### CODE REVIEW

File: actions/helm_generate_test.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	"github.com/arschles/assert"
+)
+
+func TestGenerateParamsNoStage(t *testing.T) {
+	fakeFS := getFakeFileSys()
+	stagingDir := "staging/foo"
+	defaultParamsComponentAttrs := genParamsComponentAttrs{
+		Org:        "org",
+		Tag:        "",
+		PullPolicy: "stayGangsta",
```
Comment (bacongobbler) [inline actions/helm_generate_test.go]: +1

File: actions/helm_stage_test.go [hunk 2]
```diff
@@ -143,20 +143,21 @@ func TestCreateDir(t *testing.T) {
...
-		map[string][]byte{"foo": []byte{}},
+		map[string]*bytes.Buffer{"foo": &bytes.Buffer{}},
 		"file system contents")
 }
 
 func TestCreateDirAlreadyExists(t *testing.T) {
 	fakeFileSys := getFakeFileSys()
 
-	fakeFileSys.Files["foo"] = []byte{}
+	// fakeFileSys.Files["foo"] = []byte{}
```
Comment (bacongobbler) [inline actions/helm_stage_test.go]: can you remove this comment please? :)

Comment (vdice) [inline actions/helm_stage_test.go]: ah yes will do, thanks

File: actions/helm_generate.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+// NopWriteCloser returns a WriteCloser with a no-op Close method wrapping
+// the provided Writer w.
+func NopWriteCloser(w io.Writer) io.WriteCloser {
+	return nopWriteCloser{w}
+}
+
+func generateParams(stage bool, fs fileSys, whereTo string, paramsComponentMap genParamsComponentMap) error {
+	var executeTo = NopWriteCloser(os.Stdout)
+	if stage {
```
Comment (arschles) [inline actions/helm_generate.go]: make sure to `defer executeTo.Close()` here. if it ends up being a file, you'll want to close it

**Comment by arschles** [comment]
LGTM2 assuming https://github.com/deis/deisrel/pull/24#discussion_r62579908 is done

**Pull request merged**